src/direct-auth-idx/direct-auth-idx/Views/Account/Login.cshtml

```diff
@@ -62,19 +62,23 @@
                 @Html.AntiForgeryToken()
                 <div class="form-group">
                     <div class="col-md-offset-2 col-md-10">
-                        <input type="submit" value="Forgot Password?" class="btn btn-default btn-stretch-wide"/>
+                        <input type="submit" value="Forgot Password?" class="btn btn-default btn-stretch-wide" />
                     </div>
                 </div>
             }
         </section>
 
+        <section id="idpButtons">
```

Comment:
Let's add a comment here to let developers know they can remove this piece of code if they don't want social login (as Adam suggested). Also, we might need to add similar comments in the login method.